openfisca_country_template/parameters/taxes/social_security_contribution.yaml

```diff
@@ -1,4 +1,6 @@
 # The social_security_contribution is calculated with a marginal scale. Marginal scales are represented with a specific structure.
+description: Social security contribution tax scale
+unit: currency:EUR
```

Comment:
Actually, this is ambiguous: the thresholds are in `EUR`, but the rates are unit-less! See [issue raised by france](https://github.com/openfisca/openfisca-france/issues/1019) (in FR, sorry)